routing/websocket/pod.json

```diff
@@ -0,0 +1,29 @@
+{
+  "kind": "Pod",
+  "apiVersion":"v1",
+
+  "metadata": {
+        "name": "websocket",
+        "labels": {
+                "name": "hello-websocket"
+        }
+  },
+
+  "spec": {
+      "containers": [{
+        "name": "hello-websocket",
+        "image": "pweil/hello-websocket",
```

Comment:
the hello-websocket image is only stored under pweil ? what if pweil removed it? is there any official image?